pulsar/resource_pulsar_namespace_test.go

```diff
@@ -424,3 +497,76 @@ resource "pulsar_namespace" "test" {
 }
 `, wsURL, ns)
 }
+
+func testPulsarNamespaceWithPermissionGrants1(wsURL, cluster, tenant, ns string) string {
```

Comment:
I think it is better to pass in the changes between the grants1 and two as a parameter

```
func testPulsarNamespaceWithPermissionGrants1(wsURL, cluster, tenant, ns string, grants []map[string]string) string {
```